src/validate_historic_driver_data.R

```diff
@@ -36,7 +36,7 @@ validate_historic_driver_data <- function(var, fn, data_nc, hruids, time, time_f
   # Max of total storage in all of historic data for all HRUs is ~5,000 mm
   # Considering 300 mm is about 1 ft of water
   # An absolute max for order of magnitude check of 10,000 mm seems appropriate
-  assert_that(max(data_nc) < 10000)
+  #assert_that(max(data_nc) < 10000)
```

Comment:
i commented this out, It was failing with
```
[mhines@yeti-login2 wbeep] tail -f logs/slurm_sum_storage.out -n 10000
Reading in NetCDF data for soil_moist_tot
Started tests for soil_moist_tot
Completed tests for soil_moist_tot
Reading in NetCDF data for pkwater_equiv
Started tests for pkwater_equiv
Error: max(data_nc) not less than 10000
Execution halted
srun: error: UV00000395-P001: task 0: Exited with exit code 1
```

Reply:
^ so we need to contact them about this. That check is in there because last year they gave us data that was way over the order of magnitude that was reasonable. Seems like it has happened again. Might be a units thing. I believe that is what it was last time

Reply:
hence ... https://github.com/usgs-makerspace/wbeep-processing/pull/163/files#diff-7965945489d27ea017758335b5b3f223R44-R45

Reply:
I'm checking if the other 4 after pkwater also have this issue and then will send a chat out about it

Reply:
ok, i wasn't sure because of the changes to the geospatial fabric, if any were so different that the failed checks might be possible to be that large!

Reply:
Hmmm that's true. I guess I can't be sure that it isn't right but seems unlikely that the GF would bring ~100ft increase in water on the surface!!

Reply:
Based on what you commented out in the script that is on yeti, it seems like `hru_impervstor` and `gwres_stor` were also having this issue. Is that right?

Reply:
hahah! 😱 hopefully not! 😩  are you following up with them about it on teams?

Reply:
yep, about to but just want to make sure I tell them which ones were causing problems

Reply:
ah, gotcha! once I hit this failure with pkwater_equiv i commented it out, so I'm not sure if it would have failed for any other variables

Reply:
Ok. I was going to test it out to see but I keep getting failures about the R/3.5.1 thing.

![image](https://user-images.githubusercontent.com/13220910/94621460-f3d23180-0275-11eb-9ac5-f84d8beab357.png)